### PR TITLE
fix: resource hints images

### PR DESCRIPTION
**Pull request by Zizzamia**
In this PR there are an incremental improvements related to:
- better bio
- lowercased "resource hints"
- added more code examples
- clarified that there is no standard across the browsers for Native Lazy Loading
- uploaded images for figure_markup

There are still several TODOs to take care of, but I am busy to celebrate my Belated-Birthday today so I figured was best I kick one small PR now, and tomorrow I take care of the rest of TODOs.

Have a great Sunday @rviscomi & @bazzadp 🎄

Progress on #920

### CODE REVIEW

File: src/content/en/2020/resource-hints.md [hunk 1]
```diff
@@ -8,7 +7,7 @@ authors: [Zizzamia]
 reviewers: [jessnicolet, pmeenan, giopunt, mgechev, notwillk]
 analysts: [khempenius]
 translators: []
-zizzamia_bio: Staff Software Engineer, Retail Growth at <a href="https://www.coinbase.com/">Coinbase</a>
+zizzamia_bio: Leonardo is a Staff Software Engineer at <a href="https://www.coinbase.com/">Coinbase</a>, leading web performance and growth initiatives. He curates the <a href="https://ngrome.io">NGRome Conference</a>. Leo also maintains the <a href="https://github.com/Zizzamia/perfume.js">Perfume.js<a/> library, which helps companies prioritize roadmaps and make better business decisions through performance analytics.
```
Comment (bazzadp) [inline src/content/en/2020/resource-hints.md:10]:
```suggestion
zizzamia_bio: Leonardo is a Staff Software Engineer at <a href="https://www.coinbase.com/">Coinbase</a>, leading web performance and growth initiatives. He curates the <a href="https://ngrome.io">NG Rome Conference</a>. Leo also maintains the <a href="https://github.com/Zizzamia/perfume.js">Perfume.js<a/> library, which helps companies prioritize roadmaps and make better business decisions through performance analytics.
```

File: src/content/en/2020/resource-hints.md [hunk 2]
```diff
@@ -86,54 +94,46 @@ Be mindful, depending on the [quality](https://github.com/andydavies/http2-prior
...
+```html
+<img src="image.png" loading="lazy" alt="…" width="200" height="200">
+```
 
-{# TODO - clarify that there is no standard across the browsers - Ref https://github.com/HTTPArchive/almanac.httparchive.org/pull/1587#discussion_r532286705 #}
+Be mindful Chromium's implementation of lazy-loading thresholds logic historically has been quite [conservative](https://web.dev/browser-level-image-lazy-loading/#distance-from-viewport-thresholds), keeping the offscreen limit to 3000px. During the last year the limit has been actively tested and improved on to better align developer expectations, and ultimately moving the thresholds to 1250px. Also, there is [no standard across the browsers](https://github.com/whatwg/html/issues/5408) and no ability for web developers to override the defaults thresholds provided by the browsers, yet.
```
Comment (bazzadp) [inline src/content/en/2020/resource-hints.md:101]:
```suggestion
Be mindful Chromium's implementation of lazy-loading thresholds logic historically has been quite [conservative](https://web.dev/browser-level-image-lazy-loading/#distance-from-viewport-thresholds), keeping the offscreen limit to 3000px. During the last year the limit has been actively tested and improved on to better align developer expectations, and ultimately moving the thresholds to 1250px. However, there is [no standard across the browsers at present](https://github.com/whatwg/html/issues/5408) and no ability for web developers to override the defaults thresholds provided by the browsers, yet.
```

**Review by rviscomi — APPROVED** [review]
LGTM. We can review in more detail in the subsequent edit (cc @exterkamp)

My only nit is that the data labels in the chart appear to be using a serif font, diverging from the [data viz style guide](https://docs.google.com/spreadsheets/d/1OtE2wqjGOMy7pRi3uorK7ITVDJXAVoP-r69k8eeKzeU/edit#gid=610233243). It's not a huge discrepancy so not blocking, but would be nice to correct if you have time.

**Review by bazzadp — COMMENTED** [review]
A couple of minor suggestions.

File: src/content/en/2020/resource-hints.md (hunk 1 above)
Comment (Zizzamia) [reply src/content/en/2020/resource-hints.md:10]: Sweet, thanks! Changing this in the next PR.

File: src/content/en/2020/resource-hints.md (hunk 2 above)
Comment (Zizzamia) [reply src/content/en/2020/resource-hints.md:101]: Nice! Changing this as well in the next PR.

**Comment by Zizzamia** [comment]
@rviscomi interesting, I used the links you pasted https://docs.google.com/spreadsheets/d/e/2PACX-1vTYAbLxN40s6mNR1jo0XDe_V4siN8TAsx2mryMp5IQmlJ-9O9eJxYROz7Rw6ozyFP6hlIZHxxh95GqX/pubchart?oid=1550112064&format=interactive. Alright, I am going to figure out how to fix it 😁

**Pull request merged**